src/app/modules/main/communities/controller.nim

```diff
@@ -5,25 +5,29 @@ import ../../../core/signals/types
 import ../../../core/eventemitter
 import ../../../../app_service/service/community/service as community_service
 import ../../../../app_service/service/contacts/service as contacts_service
+import ../../../../app_service/service/community_tokens/service as tokens_service
 
 type
   Controller* = ref object of RootObj
     delegate: io_interface.AccessInterface
     events: EventEmitter
     communityService: community_service.Service
     contactsService: contacts_service.Service
+    tokensService: tokens_service.Service
 
 proc newController*(
     delegate: io_interface.AccessInterface,
     events: EventEmitter,
     communityService: community_service.Service,
     contactsService: contacts_service.Service,
+    tokensService: tokens_service.Service,
     ): Controller =
   result = Controller()
   result.delegate = delegate
   result.events = events
   result.communityService = communityService
   result.contactsService = contactsService
+  result.tokensService = tokensService
```

Comment:
A minor thing, but maybe we should name this `communityTokenService` instead. We already have `tokenService` and it will be added to the communities modules as well. We'll then end up with `tokenService` and `tokensService` which is a great recipe for painful debugging sessions.

Reply:
Done